apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Snippets/SpannerConnectionSnippets.cs

```diff
@@ -242,6 +259,7 @@ public async Task CommitTimestampAsync()
 
                     // Display the inserted values
                     SpannerCommand selectCmd = connection.CreateSelectCommand("SELECT * FROM UsersHistory");
+                    selectCmd.Transaction = transaction;
```

Comment:
I don't know whether we really want to do the display part within the same transaction. It's not even clear to me whether the first two should be in the same transaction.
Aside from anything else, if they haven't been committed yet, I don't know what the commit timestamp would be... I suspect we should have two retriable bits, then execute the SELECT not in a transaction at all. That should show us two different commit timestamps.